product_default_image/views/stock_config_settings_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="view_stock_config_settings" model="ir.ui.view">
+        <field name="name">stock settings</field>
+        <field name="model">stock.config.settings</field>
+        <field name="inherit_id" ref="stock.view_stock_config_settings" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <field name="company_id" invisible="1" />
+            </xpath>
+            <xpath expr="//field[@name='group_stock_packaging']" position="after">
+                <field name="product_image_target" widget="radio" />
+		        <field name="product_default_global_image"
```

Comment:
interesting. My code editor is showing spaces with those two fields properly aligned. Looks off here though.